### PR TITLE
Add tests for break in loop/for/while loops

### PR DESCRIPTION
**Pull request by camchenry**
This adds some tests for covering the `break` expression in `for`, `while` and `loop` blocks/expressions (per #54). This is my first contribution to this project, so please let me know what I can change to increase the code quality.

### CODE REVIEW

File: tests/mod.rs [hunk 1]
```diff
@@ -89,3 +89,8 @@ fn paths_coverage() {
 fn futures_coverage() {
     check_percentage("futures", 1.0f64, true);
 }
+
+#[test]
+fn breaks_expr_coverage() {
+    check_percentage("breaks", 1.0f64, true);
+}
```
Comment (rye) [inline tests/mod.rs]: Missing a newline at the end of the file.

File: tests/data/breaks/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    };
+
+    let z = 'loop4: loop {
+        'loop5: loop {
+            break 'loop4 x;
+        }
+    };
+
+    assert_eq!(x, z);
+}
```
Comment (rye) [inline tests/data/breaks/src/lib.rs]: Missing a newline at the end of the file.

File: tests/mod.rs (hunk 1 above)
Comment (camchenry) [reply tests/mod.rs]: Good call, thanks.

**Comment by xd009642** [comment]
LGTM! Thanks for the work :+1:

**Pull request merged**